tests/requirements/py3.txt

```diff
@@ -15,6 +15,7 @@ python-memcached >= 1.59
 pytz
 pywatchman; sys.platform != 'win32'
 PyYAML
+redis >= 3.0.0
```

Comment:
Do we need the version specification? 

**Update**: Why 3.0.0? Latest is 3.5.3... 🤔 — 3.0.0 was a while ago now.) I see this was @ngnpope's comment from above: https://github.com/django/django/pull/14437#pullrequestreview-692861563 — tests do pass with that. (I'll resolve this.)